File: samples_generated_code/sample_1/qwen/baseline_code_c/code_row_77.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <time.h>

void floatToString(float number, char *str) {
    sprintf(str, "%.6f", number);
}

int main() {
    srand(time(NULL));

    float a = (float)rand() / RAND_MAX;
    float b = (float)rand() / RAND_MAX;
    float c = (float)rand() / RAND_MAX;

    char str_a[20];
    char str_b[20];
    char str_c[20];

    floatToString(a, str_a);
    floatToString(b, str_b);
    floatToString(c, str_c);

    return 0;
}

File: samples_generated_code/sample_1/phi4/baseline_code_java/code_row_77.java
import java.util.Random;

public class RandomFloatsToString {
    public static void main(String[] args) {
        Random rand = new Random();

        float randomA = rand.nextFloat();
        float randomB = rand.nextFloat();
        float randomC = rand.nextFloat();

        String str_a = convertToDecimalString(randomA);
        String str_b = convertToDecimalString(randomB);
        String str_c = convertToDecimalString(randomC);

        System.out.println("str_a: " + str_a);
        System.out.println("str_b: " + str_b);
        System.out.println("str_c: " + str_c);
    }

    public static String convertToDecimalString(float number) {
        return String.format("%.6f", number); // Adjust the precision as needed
    }
}